tox.ini

```diff
@@ -33,3 +33,4 @@ per-file-ignores =
 ignore =
     fasm/parser/tags.py
     fasm/parser/libparse_fasm.*
+    third_party/antlr4/runtime/PHP
```

Comment:
Should probably ignore `third_party/antlr4` completely?

Reply:
I'll add a comment here, but the reason this is needed specifically is because this is an empty directory, and this causes problems: https://stackoverflow.com/questions/2640378/include-empty-directory-with-python-setup-py-sdist

The typical workaround is to just put a file in there, but this is in a submodule.

Reply:
Putting `third_party` here didn't work either.

Reply:
That seems fine.  Add a comment explaining that.

Reply:
Done.